README.md

```diff
@@ -69,6 +77,36 @@ lock.on("authenticated", function(authResult) {
 });
 ```
 
+### new Auth0LockPasswordless(clientID, domain, options)
+
+Initializes a new instance of `Auth0LockPasswordless` configured with your application `clientID` and your account's `domain` at [Auth0](https://manage.auth0.com/). You can find this information at your [application settings](https://manage.auth0.com/#/applications).
```

Comment:
"You can find this information at your ..." => "You can find this information in your ..."